locations/spiders/mma_fr.py

```
@@ -0,0 +1,33 @@
+    name = "mma_fr"
+    item_attributes = {"brand": "MMA", "brand_wikidata": "Q3331046"}
+    sitemap_urls = ["https://agence.mma.fr/home.sitemap.xml"]
+    wanted_types = ["InsuranceAgency"]
+    drop_attributes = ["facebook", "image"]
+
+    def post_process_item(self, item, response, ld_data, **kwargs):
+        item["branch"] = (item.pop("name", "") or "").removeprefix("AGENCE D'ASSURANCE MMA ")
+        try:
+            item["opening_hours"] = self.parse_opening_hours(ld_data)
+        except Exception:
+            pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)